inc/xuc.class.php

```diff
@@ -169,12 +201,22 @@ function getUserInfosByPhone($params = []) {
       } elseif (count($data['users']) == 1) {
          $current_user     = current($data['users']);
          $users_id         = $current_user['id'];
-         $iterator_tickets = $DB->request("SELECT glpi_tickets.id, glpi_tickets.name, glpi_tickets.content
-                                           FROM glpi_tickets
-                                           INNER JOIN glpi_tickets_users
-                                             ON glpi_tickets_users.tickets_id = glpi_tickets.id
-                                             AND glpi_tickets_users.type = ".CommonITILActor::REQUESTER."
-                                           WHERE glpi_tickets.status < ".CommonITILObject::SOLVED);
+         $iterator_tickets = $DB->request([
+            'SELECT'     => ['glpi_tickets.id', 'glpi_tickets.name', 'glpi_tickets.content'],
+            'FROM'       => 'glpi_tickets',
+            'INNER JOIN' => [
+               'glpi_tickets_users' => [
+                  'FKEY' => [
+                     'glpi_tickets_users' => 'tickets_id',
+                     'glpi_tickets'       => 'id',
+                  ]
+               ]
+            ],
+            'WHERE'     => [
+               'glpi_tickets_users.type' => CommonITILActor::REQUESTER,
+               'glpi_tickets.status'     => ["<=", CommonITILObject::SOLVED],
```

Comment:
Was `<` not `<=` in the original query. Is that inteded?

Reply:
nop